notebook/i18n/de/LC_MESSAGES/notebook.po

```diff
@@ -0,0 +1,416 @@
+"Language: de\n"
+"X-Generator: Poedit 1.8.11\n"
+
+msgid "The Jupyter Notebook requires tornado >= 4.0"
+msgstr "Der Jupyter Notizblock erfordert Tornado >= 4.0"
+
+msgid "The Jupyter Notebook requires tornado >= 4.0, but you have < 1.1.0"
+msgstr "Der Jupyter Notizblock erfordert Tornado >= 4.0 aber Sie < 1.1.0 haben"
+
+msgid "The Jupyter Notebook requires tornado >= 4.0, but you have %s"
+msgstr "Der Jupyter Notizblock erfordert Tornado >= 4.0 aber Sie %s haben"
+
+msgid "The `ignore_minified_js` flag is deprecated and no longer works."
+msgstr ""
+"Die `ignore_minified_js `Flagge wird missbilligt und funktioniert nicht mehr."
+
+msgid ""
+"Alternatively use `%s` when working on the notebook's Javascript and LESS"
+msgstr ""
+"Alternativ verwenden Sie `%s, `wenn auf dem Javascript des Notizblocks zu "
+"funktionieren und weniger"
+
+msgid ""
+"The `ignore_minified_js` flag is deprecated and will be removed in Notebook "
+msgstr ""
+"Die `ignore_minified_js `Flagge wird missbilligt und wird entfernt in "
+"Notizblock 6.0 sein"
+
+msgid ""
+"Widgets are unavailable. Please install widgetsnbextension or ipywidgets 4.0"
+msgstr ""
+"Dinger sind nicht verfügbar. Bitte installieren Sie widgetsnbextension oder "
+"ipywidgets 4.0"
+
+msgid "List currently running notebook servers."
+msgstr "Listen Sie auf und gegenwärtig führen Sie Notizblockserver aus."
+
+msgid "Produce machine-readable JSON output."
+msgstr "Stellen Sie maschinenlesbare JSON Ausgabe her."
+
+msgid ""
+"If True, each line of output will be a JSON object with the details from the "
+"server info file."
+msgstr ""
+"Wenn jede Zeile von Ausgabe wahr ist, wird sie ein JSON Objekt mit den "
+"Details von der Serverinformationsdatei sein."
+
+msgid "Currently running servers:"
+msgstr "Gegenwärtig Server ausführen:"
+
+msgid "Don't open the notebook in a browser after startup."
+msgstr "Öffnen Sie den Notizblock in einem Browser nicht nach Startup."
+
+msgid ""
+"DISABLED: use %pylab or %matplotlib in the notebook to enable matplotlib."
+msgstr ""
+"Unfähig gemacht: Verwenden Sie %pylab oder %matplotlib im Notizblock, um "
+"matplotlib zu ermöglichen."
+
+msgid "Allow the notebook to be run from root user."
+msgstr "Erlauben Sie, dass der Notizblock von Root geführt wird."
+
+msgid ""
+"The Jupyter HTML Notebook.\n"
+"    \n"
+"    This launches a Tornado based HTML Notebook Server that serves up an "
+"HTML5/Javascript Notebook client."
+msgstr ""
+"Die Jupyter HTML Notebook.n n dies startet einen tornadobasierten HTML "
+"Notizblockserver, der einen HTML5/ Javascript Notizblockkunden hinauf dient."
+
+msgid ""
+"Deprecated: Use minified JS file or not, mainly use during dev to avoid JS "
+msgstr ""
+"Missbilligt: Verwenden Sie minified JS Akte oder nicht vermeiden Sie während "
+"dev hauptsächlich gewöhnlich JS recompilation"
+
+msgid "Set the Access-Control-Allow-Credentials: true header"
+msgstr ""
+"Setzen Sie die Zugangskontrolle erlauben Sie Ausweispapiere: Wahre Überschrift"
```

Comment:
`Access-Control-Allow-Credentials` shouldn't be translated again - the HTTP protocol uses English words whatever the locale of the user is.

I know the translations aren't the focus here, but if we're going to merge it with German translations in, I think it's worth doing a bit of basic QA.